src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -58,6 +60,10 @@ class MetadataBasedItemList extends React.Component<Props, State> {
         };
     }
 
+    getMetadataColumnName(column: MetadataColumnConfig | string): string {
+        return typeof column === 'string' ? column : getProp(column, 'name');
```

Comment:
Do we need to add a `defaultValue` for `getProp`?

Reply:
These are columns passed by the parent app. We don't know what could be the default value.